docs/next/guides/building-and-testing/building.md

```diff
@@ -9,83 +9,263 @@ tags:
+Each Handsontable [project](#about-building) has its own [building](#building-the-packages) processes defined in its own `package.json` file. Apart from that, the root directory has its own `package.json` file as well:
 
-## NPM tasks for the framework-specific wrappers.
+| File                              | Holds tasks for building:                           |
+| --------------------------------- | --------------------------------------------------- |
+| `./package.json`                  | - All the packages at once<br>- Individual packages |
+| `./handsontable/package.json`     | The JavaScript package                              |
+| `./wrappers/angular/package.json` | The Angular package                                 |
+| `./wrappers/react/package.json`   | The React package                                   |
+| `./wrappers/vue/package.json`     | The Vue package                                     |
 
-Each of the wrappers has its own framework-specific build environment and scripts. The build process can be triggered by either navigating the the wrapper directory (e.g. `./wrappers/react-handsontable/` and running:
+## Running your first build
 
-*   `npm run build`
+To run your first build:
+1. Install [Node.js](https://nodejs.org/) (version **15.11**+).
+2. Install [npm](https://www.npmjs.com/) (version **7.17**+).
+3. Clone the [Handsontable repository](https://github.com/handsontable/handsontable).
+4. From the root directory, run `npm install`.<br>All the required dependencies get installed.
+5. From the root directory, run `npm run build`.<br>All the Handsontable packages get built.
 
-Or by utilizing one of the helper scripts in the root directory:
+## Building the packages
 
-*   `npm run in` - Runs a command for the specified project. For Example:  
-    `npm run in angular build` - runs the `build` command for the `@handsontable/angular` project.  
-    Shorthands are also available - the aforementioned command works as `npm run in angular build` as well.
-*   `npm run all` - Runs a command for all the packages. For example:  
-    `npm run all build` will run the `build` command for all the defined packages.  
-    The order in which the packages are executed is defined in the `./scripts/run-all.mjs` script.
+You can either build all the packages at once, or build each package individually.
 
-## Running your first build
+### Building all the packages
+
+To build all the packages at once:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to the root directory.
+3. Run `npm run build`.<br>The script builds the following packages:
+     - The JavaScript package
+     - The Angular package
+     - The React package
+     - The Vue package
+     - A code examples package
+
+#### Root directory build tasks
+
+You can also run individual root directory `build` tasks:
+
+::: details Root directory build tasks
+`build:codesandbox`
+  - PLACEHOLDER
```

Comment:
This one is a `build` command for the Codesandbox CI, I don't think it's usable to anyone but the CI itself, so I'd skip this entire section.

Reply:
Done: d03812f